conversation/src/test/java/com/ibm/watson/developer_cloud/conversation/v1/ConversationServiceIT.java

```diff
@@ -743,20 +743,13 @@ public void testUpdateExample() {
     service.createExample(createOptions).execute();
 
     try {
-      Mentions mentions = new Mentions();
-      mentions.setEntity("entity");
-      mentions.setLocation(Arrays.asList(0L, 10L));
-      List<Mentions> mentionsList = new ArrayList<>();
-      mentionsList.add(mentions);
       UpdateExampleOptions updateOptions = new UpdateExampleOptions.Builder(workspaceId, exampleIntent, exampleText)
           .newText(exampleText2)
-          .newMentions(mentionsList)
           .build();
       Example response = service.updateExample(updateOptions).execute();
       assertNotNull(response);
       assertNotNull(response.getExampleText());
       assertEquals(response.getExampleText(), exampleText2);
-      assertEquals(response.getMentions(), mentionsList);
```

Comment:
Any reason you are removing mentions here and in Assistant? Are these duplicates?